rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -149,6 +150,20 @@
   }
 
   /**
+   * Create an observable of {@link ViewTreeObserverDrawEvent} for draws on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link ViewTreeObserver#addOnDrawListener} to observe
+   * draws. Multiple observables can be used for a view at a time.
+   */
+  @CheckResult @NonNull
+  public static Observable<ViewTreeObserverDrawEvent> drawEvents(@NonNull View view) {
```

Comment:
Do we use `Object` elsewhere? I'm having a hard time remembering, but I'm pretty sure `Void` and emitting `null` is the canonical "signal" observable as neither the type nor the value is useful.

If we use `Object` already just use that, and I'll change them all to `Void` at once.